sources/packages/api/src-sql/Reports/Create-disbursement-report.sql

```diff
@@ -0,0 +1,44 @@
+--Insert config for disbursement report.
+INSERT INTO
+    sims.report_configs (report_name, report_sql)
+VALUES
+    (
+        'Disbursement_Report',
+        'SELECT "Date of Disbursement",
```

Comment:
Is there a reason to add this select on the top of the 2 queries?

Reply:
Yes, as we are following the report generation using 3 params -> offering intensity, start date and end date of disbursement. When we create 2 where clause inside the union queries, it is expecting 6 params, 3 for the first set and 3 for the second. So I tested the scenarios, with the where clause inside and outside and the results are the same. So created the overall select at the top.

Reply:
As discussed changed